resources/Makefile

```diff
@@ -1,5 +1,5 @@
 COMPUTE_CAPABILITY=compute_70
-CUDA_VER=10.2
+CUDA_VER=11.8
```

Comment:
It seems unlikely that we need a newer cuda versions and a higher compute compability for these changes (-> huge diff). Would you mind recompiling with the old 10.2 nvcc?